docs/graphql/manual/guides/integrations/auth0-jwt.rst

```diff
@@ -43,6 +42,46 @@ In the dashboard, navigate to ``Rules``. Add the following rules to add our cust
       callback(null, user, context);
     }
 
+Test auth0 login and generate sample JWTs for testing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You don't need to integrate your UI with auth0 for testing. You call follow the steps below:
+
+1. Login to your auth0 app by heading to this URL: ``https://<auth0-domain>.auth0.com/login?client=<client_id>&protocol=oauth2&response_type=token%20id_token&redirect_uri=<callback_uri>&scope=openid%20profile``
+
+   - Replace ``<auth0-domain>`` with your auth0 app domain.
+   - Replace ``callback_uri`` with ``https://localhost:3000/callback`` or the URL you entered above. Note that this URL doesn't really need to exist while you are testing.
```

Comment:
Add another point about replacing the client_id with the correct value. Screenshot shows it though.